e2e/academic-honesty.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { execSync } from 'child_process';
+import { test, expect } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+test.describe('When the user has not accepted the Academic Honesty Policy', () => {
+  test.use({ storageState: 'playwright/.auth/development-user.json' });
+
+  test.beforeAll(() => {
+    execSync('node ./tools/scripts/seed/seed-demo-user');
+  });
+
+  test('Should validate Academy Honesty Settings', async ({ page }) => {
+    await page.goto('/settings');
+    await expect(
+      page.getByRole('heading', {
+        name: translations.settings.headings.honesty
+      })
+    ).toBeVisible();
+    await expect(
+      page.getByText(translations.settings.honesty.p1)
+    ).toBeVisible();
+    await expect(
+      page.getByText(translations.settings.honesty.p2)
+    ).toBeVisible();
+    await expect(
+      page.getByText(translations.settings.honesty.p3)
+    ).toBeVisible();
+    await expect(
+      page.getByText(translations.settings.honesty.p4)
+    ).toBeVisible();
+    await expect(
+      page.getByText(translations.settings.honesty.p5)
+    ).toBeVisible();
+    await expect(
+      page.getByText(translations.settings.honesty.p6)
+    ).toBeVisible();
+    const agreeButton = page.getByRole('button', {
+      name: translations.buttons['agree-honesty']
+    });
+    await expect(agreeButton).toBeVisible();
```

Comment:
```suggestion
```
It's checked before `click`ing, so we don't need a specific assertion here. See [their docs](https://playwright.dev/docs/actionability#introduction)